internal/util/util.go

```diff
@@ -0,0 +1,56 @@
+// Copyright © 2021 The systemk authors
```

Comment:
please rename `util` to something more meaningful

Reply:
Suggestions? These are utils used in code scattered in other packages, including tests.